{{cookiecutter.project_name}}/{{cookiecutter.project_slug}}/cli.py

```diff
@@ -6,7 +6,7 @@
 
 
 def parse_args(args):
-    """Returns parsed commandline arguments.
+    """Returns parsed command line arguments.
```

Comment:
:+1:

Reply:
As discussed off-line. Keeping this change in this PR